packages/sdk/contractkit/src/wrappers/Accounts.ts

```diff
@@ -437,6 +437,33 @@ export class AccountsWrapper extends BaseWrapper<Accounts> {
    */
   setMetadataURL = proxySend(this.connection, this.contract.methods.setMetadataURL)
 
+  /**
+   * Set a validator's payment delegation settings.
+   * @param beneficiary The address that should receive a portion of validator
+   * payments.
+   * @param fraction The fraction of the validator's payment that should be
+   * diverted to `beneficiary` every epoch, given as FixidityLib value. Must not
+   * be greater than 1.
+   * @dev Use `deletePaymentDelegation` to unset the payment delegation.
+   */
+  setPaymentDelegation = proxySend(this.connection, this.contract.methods.setPaymentDelegation)
```

Comment:
Could we see some tests in `Accounts.test.ts` for the added functions?

Reply:
Done! Added tests to confirm invalid fraction, that set values persist, and that deleted delegation really clears